src/components/layout/Footer/Footer.tsx

```
@@ -13,8 +13,13 @@ export const Footer = (): React.ReactElement => {
               <Icon name="fa-feather" size="lg" ariaLabel="Paperlyte logo" />
               <span className={styles.logoText}>Paperlyte.</span>
             </div>
-            <p className={styles.tagline}>Designed for clarity in a chaotic world.</p>
-            <p className={styles.copyright}>© {currentYear} All rights reserved.</p>
+            <p className={styles.tagline}>Your thoughts, unchained.</p>
+            <p className={styles.copyright}>
+              © {currentYear} Paperlyte. All rights reserved.
+            </p>
+            <p className={styles.taglineSecondary}>
+              Built with <span aria-label="love">💙</span> for people who think fast.
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)